cornflow-dags/DAG/rostering/core/experiment.py

```diff
@@ -127,3 +133,40 @@ def check_skills_demand(self) -> list:
                 lambda v: {"timeslot": v[0], "id_skill": v[1], "number_missing": v[2]}
             )
         )
+
+    def get_one_employee_percentage(self) -> float:
+        """Returns the percentage of time slots where only one employee is working"""
+        ts_employee = self.solution.get_ts_employee()
+        return (
+            sum(1 for e in ts_employee.values() if len(e) == 1) / len(ts_employee) * 100
```

Comment:
Although ts_employee coincides with the total number of time slots, I would divide by the total number of timeslots just in case there is no need to always have an employee

Reply:
But then the timeslot should not be created, no? As the list of timeslots should be just the timeslots where there has to be _someone_ working

Reply:
Of course, that's why the values are the same. But if the requirement were to change and instead of there always having to be someone only needed to cover the demand, the timeslot could be created, it could be necessary to cover the minimum number of hours

Reply:
Ok, will change it, it should be an easy change